### PR TITLE
Converted all activations except heaviside function

### LINKED ISSUE #60: Convert Activations from simple functions to classes and add gradient method.
Currently, all the activations in activations.py are simple functions. 

However, for future implementation of Neural Networks, we will also need derivatives methods of each of these
functions. You can have a look into loss_func.py for reference. There, each class represents a loss function and has both loss
and derivative methods. You have to implement something similar for activations.

### PR DESCRIPTION
**Pull request by Abjcodes**
#### References to other Issues or PRs or Relevant literature

Fixes #60 

#### Brief description of what is fixed or changed
Created Softmax, Softsign, Elu, and Swish classes with activation and derivative methods.

### CODE REVIEW

**Comment by kwanit1142** [comment]
@Abjcodes, please fix some issues, detected by Travis-CI, follow the PEP-8 format for syntax writing.

**Comment by Abjcodes** [comment]
Hey @kwanit1142 , Can you review the PR?

**Comment by agrawalshubham01** [comment]
@Abjcodes kindly run the tests `python3 -m pytest --doctest-modules --cov=./ --cov-report=html`. Also add `tests` as well.

**Comment by Abjcodes** [comment]
> @Abjcodes kindly run the tests `python3 -m pytest --doctest-modules --cov=./ --cov-report=html`. Also add `tests` as well. There are errors.

I will look into it.

**Comment by agrawalshubham01** [comment]
@kwanit1142 I have solved merge conflict, will be merging after tests pass!! Rest this pr LGTM.

**Review by agrawalshubham01 — APPROVED** [review]
